src/tutorials/0006-anatomy-of-a-cid/05.md

```diff
@@ -0,0 +1,32 @@
+---
+    title: "CIDv1: Multibase prefix"
+    type: "multiple-choice"
+---
+
+So now our CIDv1 in binary looks like this:
+
+`<cid-version><ipld-format><multihash>`
+
+Since binary CIDs are not very human-friendly, we can represent these binary CIDs in a string form (binary data represented as text). Example:
+
+`bafybeigdyrzt5sfp7udm7hu76uh7y26nf3efuylqabf3oclgtqy55fbzdi`
+
+Converting data between binary and string formats requires base encoding, so when working with string CIDs it's important that we know the type of base encoding that was applied to the binary data. But how can we identify this?
+
+In `CIDv0`, hashes are **always** encoded with `base58btc`. _Always_. This means that we can safely interpret `CIDv0` hashes assuming they are using `base58btc`. But, for performance reasons, we may want to support multiple base encodings. For that, you guessed it, we can add another prefix!
+
+## Multibase prefix
+
+[Multibase](https://github.com/multiformats/multibase) prefixes, which represent the base encoding used when converting CIDs between string and binary formats, are **only used in the string form of the CID**:
+
+![Multibase Prefix](tutorial-assets/T0006L05-multibase-prefix.jpg)
+
+Let's examine two examples of CIDs in their string form:
+
+![String form examples](tutorial-assets/T0006L05-string-form.jpg)
+
+We know the first one is a `CIDv0` because it starts with `Qm...`. All hashes that start with `Qm` can safely be interpreted as `base58btc` as a CID of Version 0.
```

Comment:
```suggestion
We know the first one is a `CIDv0` because it starts with `Qm...`. All CIDs that start with `Qm` can safely be interpreted as `base58btc` as a CID of Version 0.
```